include/picongpu/fields/currentDeposition/EmZ/DepositCurrent.hpp

```diff
@@ -80,18 +80,31 @@ namespace picongpu
                 {
                     if(line.m_pos0[2] == line.m_pos1[2])
                         return;
+
+                    auto const shape_i = make_TrajectoryAssignmentShapeFunction(
```

Comment:
I think here `x/y/z` would be better name than `i/j/k` as this is actually a function that describes that shape. And later on you call it with `i/j/k` to get the actual value for those indices.

If you decide to change, the same applies to naming of types inside strategy.

Also, why not use our normal camelCase?

Reply:
I will keep `i/j/k` because the loops do not reflect or naming of the directions `x/y/z`

Reply:
I will keep `Shape_i`, `Sape_j` with `-` for now because using camel case  `ShapeI`, ShapeJ` is depending on the font not readable.
Is the last letter a capital `I` or a small `l`